internal/flow/flow.go

```diff
@@ -211,6 +212,20 @@ func Promote(ctx context.Context, configRepoURL, artifactFileName, service, env,
 	if err != nil {
 		return "", errors.WithMessage(err, fmt.Sprintf("commit changes from path '%s'", destinationPath))
 	}
+	err = notifyRelease(slack.ReleaseOptions{
+		SlackToken:    slackToken,
+		Service:       service,
+		Environment:   env,
+		ArtifactID:    sourceSpec.ID,
+		CommitAuthor:  sourceSpec.Application.AuthorName,
+		CommitMessage: sourceSpec.Application.Message,
+		CommitSHA:     sourceSpec.Application.SHA,
+		CommitLink:    sourceSpec.Application.URL,
+		Releaser:      committerName,
+	})
+	if err != nil {
+		log.Errorf("flow; promote; error notifying release: %v", err)
```

Comment:
Usually we use colon as separators

```suggestion
		log.Errorf("flow: Promote: error notifying release: %v", err)
```